setup_cmake.ch

```diff
@@ -8,5 +8,6 @@ travis_retry wget --no-check-certificate -O - $CMAKE_URL | tar --strip-component
 cd cmake-$CMAKE_VERSION
 ./bootstrap
 make && sudo make install && cd ..
+rm -rf cmake-$CMAKE_VERSION.tar.gz
```

Comment:
This belongs to the same commit

Reply:
FYI, we do squash commits for you. Please ensure that your commits are ready to be merged.

Reply:
i got it